### DISCUSSION TITLE
Technical Writer Employment Links Are Failing

### DISCUSSION THREAD

**Discussion by bkerrbske** [community/community#148202] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Greetings,

I tried to apply to the Staff Technical Writer role from Indeed.com, however, the link fails with the following error after clicking to allow access to my Indeed account with the following:

 Error: Unable to log in with the social account. Please make sure your iCIMS account is connected to one before logging in. 

The same results for other Technical Writer roles as well.  I even created an account with GitHub and the same results.  Please advise so I can apply for the roles that align with my background.

Thank you,
Brian

**Comment by jakubbbdev2** [comment]
Hi @bkerrbske 

It seems you're having trouble applying for Technical Writer roles via Indeed due to an iCIMS login error. Here are some steps to resolve this:

Check iCIMS Account: Ensure your iCIMS account is connected to your social accounts (Indeed, GitHub).
Clear Cache: Clear your browser's cache and cookies.
Try a Different Browser: Use a different browser to see if the issue persists.
Contact Support: Reach out to iCIMS support for assistance.
Alternatively, try applying directly through the company's career page.

**Comment by divyamlathiya** [comment]
It seems you're encountering an issue with logging in via social accounts (like GitHub) on Indeed.com when trying to apply for roles. The error suggests that your iCIMS account, which is often used by companies to manage job applications, isn't connected to the social login method you're trying to use.

## Here's what you can try to fix the issue:
**1. Check iCIMS Account Connection:**  Ensure that your ``iCIMS`` account (if you have one) is properly connected to your social account (GitHub, LinkedIn, etc.). You may need to log in to iCIMS directly and connect the accounts in the settings.

**2. Clear Browser Cache/Cookies:**  Sometimes, caching issues can cause login problems. Try clearing your browser's cache or using a different browser.

**3. Use a Direct Email and Password Login:**  Instead of using the social login option, try registering or logging in with your email and a password for ``iCIMS`` (if that option is available) rather than using GitHub or another social account.

**4. Try Incognito Mode or Different Browser:**  Sometimes browser extensions or settings interfere with the login process. Try applying in an incognito window or use a different browser.

**5. Contact Indeed Support:**  If none of the above works, you may need to contact Indeed’s support team for assistance with this specific issue.

I hope one of these steps helps you apply for the roles! Let me know if you need further assistance.

## Try the following:

**1. Ensure your iCIMS account is connected to your social login (GitHub, LinkedIn, etc.).**
**2. Clear browser cache or use a different browser.**
**3. Use direct email/password login instead of social login.**
**4. Try in incognito mode or a different browser.**
**5. Contact Indeed support for further assistance.**
